ci/pipeline.yml

```diff
@@ -213,23 +221,23 @@ resources:
 - name: bats
   type: git
   source:
-    uri: git@github.com:jianqiu/bosh.git
-    branch: 256.x-softlayer
+    uri: git@github.com:jianqiu/bosh-acceptance-tests.git
```

Comment:
For stemcell, currently, we still have to fork due to the following reasons:

Need to enable root ssh remote login, both Bluemix and eCPI require it.
For Bluemix stemcell, we need to install openssl 1.0.2 and fips module.